samples/mac/weather/MyManagedStuff/MyManagedStuff/WeatherFetcher.cs

```diff
@@ -2,7 +2,7 @@
 using System.Json;
 using System.Net;
 
-namespace MyManagedStuff {
+namespace XAM {
```

Comment:
That will still look weird `XAM_ WeatherFetcher` ;-) The only way to make it look native (right now) is to avoid namespaces and prefix the type itself.

You might want to add a comment why you used a _weird_ managed name too